lib/openstudio-standards/prototypes/common/objects/Prototype.Model.exterior_lights.rb

```diff
@@ -47,37 +67,61 @@ def model_add_typical_exterior_lights(model, exterior_lighting_zone_number, onsi
     else
       ext_lights_sch_facade_and_landscape = model.alwaysOnDiscreteSchedule
     end
-    if !exterior_lighting_properties['occupancy_setback_reduction'].nil? && (exterior_lighting_properties['occupancy_setback_reduction'] > 0.0)
+
+    occupancy_setback_reduction = 0.0
+    ext_lights_sch_other = model.alwaysOnDiscreteSchedule
+    if exterior_lighting_properties.has_key?('occupancy_setback_reduction')
+      if !exterior_lighting_properties['occupancy_setback_reduction'].nil? && (exterior_lighting_properties['occupancy_setback_reduction'] > 0.0)
+        occupancy_setback_reduction = exterior_lighting_properties['occupancy_setback_reduction']
+      end
+    else
+      unless ['90.1-2004', '90.1-2007', 'DOE Ref Pre-1980', 'DOE Ref 1980-2004', 'DEER 1985', 'DEER 1996', 'DEER 2003', 'DEER 2007', 'DEER Pre-1975'].include?(template)
```

Comment:
make an inclusive criteria, not an exclusive one. There are several other standards ignored here.

Reply:
Which ones are missing? I can certainly do that but there are a lot of applicable ones as well.

Reply:
other DEER versions, the NZE templates. I'm wondering if this should go back to being something in the standard fields? If it toggles on the standard, it should probably go in there.